sqlalchemy_history/utils.py

```diff
@@ -133,10 +133,7 @@ def version_class(model):
     :returns:
     """
     manager = get_versioning_manager(model)
-    try:
-        return manager.version_class_map[model]
-    except KeyError:
-        return model
+    return manager.version_class_map.get(model, None)
```

Comment:
In relationship_builder we have
```
        try:
            self.remote_cls = version_class(self.property.mapper.class_)
            self.versioned = True
        except (AttributeError, KeyError):
            return
```
Now we won't get KeyError, We would want to make the changes over there as well.